backend/qemu.pm

```diff
@@ -418,8 +418,15 @@ sub start_qemu {
     my $qemubin = $ENV{QEMU};
 
     my $qemuimg = find_bin('/usr/bin/', qw(kvm-img qemu-img));
+
     unless ($qemubin) {
-        $qemubin = find_bin('/usr/bin/', $vars->{QEMU} ? ('qemu-system-' . $vars->{QEMU}) : qw(kvm qemu-kvm qemu qemu-system-x86_64 qemu-system-ppc64));
+        if ($vars->{QEMU}) {
+            $qemubin = find_bin('/usr/bin/', 'qemu-system-' . $vars->{QEMU});
+        }
+        else {
+            (my $class = $vars->{WORKER_CLASS}) =~ s/qemu_/qemu-system\-/g;
```

Comment:
won't this cause warnings on undefined?

Reply:
@coolo 

Yes it will. :-)

Hence the 
> (split(/\s*,\s*/, $class // '')

in the next line

Reply:
@coolo true.

Fixed.